tables/waf_traffic_log/waf_traffic_log.go

```diff
@@ -0,0 +1,130 @@
+		"captcha_response":               "The CAPTCHA action status for the request, populated when a CAPTCHA action is applied to the request. This field is populated for any CAPTCHA action, whether terminating or non-terminating. If a request has the CAPTCHA action applied multiple times, this field is populated from the last time the action was applied.",
+		"format_version":                 "The format version for the log.",
+		"http_request":                   "The metadata about the request.",
+		"http_source_id":                 "The ID of the associated resource.",
+		"http_source_name":               "The source of the request. Possible values: CF for Amazon CloudFront, APIGW for Amazon API Gateway, ALB for Application Load Balancer, APPSYNC for AWS AppSync, COGNITOIDP for Amazon Cognito, APPRUNNER for App Runner, and VERIFIED_ACCESS for Verified Access.",
+		"labels":                         "The labels on the web request. These labels were applied by rules that were used to evaluate the request. AWS WAF logs the first 100 labels.",
+		"non_terminating_matching_rules": "The list of non-terminating rules that matched the request. Each item in the list contains action, ruleId, and ruleMatchDetails.",
+		"rate_based_rule_list":           "The list of rate-based rules that acted on the request. For information about rate-based rules, see Using rate-based rule statements in AWS WAF.",
+		"request_headers_inserted":       "The list of headers inserted for custom request handling.",
+		"rule_group_list":                "The list of rule groups that acted on this request, with match information.",
+		"terminating_rule_id":            "The ID of the rule that terminated the request. If nothing terminates the request, the value is Default_Action.",
+		"terminating_rule_match_details": "Detailed information about the terminating rule that matched the request. A terminating rule has an action that ends the inspection process against a web request. Possible actions for a terminating rule include Allow, Block, CAPTCHA, and Challenge. During the inspection of a web request, at the first rule that matches the request and that has a terminating action, AWS WAF stops the inspection and applies the action. The web request might contain other threats, in addition to the one that's reported in the log for the matching terminating rule.",
+		"terminating_rule_type":          "The type of rule that terminated the request. Possible values: RATE_BASED, REGULAR, GROUP, and MANAGED_RULE_GROUP.",
+		"timestamp":                      "The date and time when the request was made, in ISO 8601 format.",
+		"web_acl_id":                     "The GUID of the web ACL.",
+
+		// Override table specific tp_* column descriptions
+		"tp_akas":      "List of ARNs (Amazon Resource Names) associated with the event, if applicable.",
```

Comment:
Are there usual ARNs we add (when available)?

Reply:
Yes, if the request includes the resource's ARN, we add that value to `tp_akas`, as it is specifically meant to store resource ARNs.